lesson_21/homework_21.py

```diff
@@ -0,0 +1,52 @@
+import time
+from datetime import datetime
+from time import gmtime, strftime
+import logging
+import re
+
+"""Засобами автоматизації проаналізуйте наданий нам лог: hblog.txt
+#відберіть лише строки з вказаним ключем Key TSTFEED0300|7E3E|0400
+Створіть функцію, що поверне лог-файл, де буде аналіз правильності вимог:
+для кожного випадку де heartbeat більше 31 сек але менше 33 логувало WARNING в файл hb_test.log
+для кожного випадку де heartbeat більше рівно 33 логувало ERROR в файл hb_test.log
+3.Зверніть увагу, що нам для аналізу помилок було б добре знати час, в який помилка відбулася.
+Обов’язково включіть результат роботи — файл hb_test.log в PR."""
+
+def find_timestamps(text_doc):
+    logging.basicConfig(
+            filename='hb_test.log',
```

Comment:
All hardcodes please try to move to the top as constants or hide within main block